vim/profiling.md

```diff
@@ -0,0 +1,37 @@
+# Vim Profiling
+
+I recently encountered a situation where Vim was taking a *very* long time to
```

Comment:
I don't like the word "very" (Mark Twain and I have this in common). How about "excruciatingly" or "noticeably" or "annoyingly" instead?